README.md

```diff
@@ -92,6 +92,30 @@ http-server --proxy http://localhost:8080?
 
 Note the `?` at the end of the proxy URL. Thanks to [@houston3](https://github.com/houston3) for this clever hack!
 
+## Using SSL
+
+First, make sure that you have `key.pem` and `cert.pem` files. You can generate them using this command:
+
+``` sh
+openssl req -newkey rsa:2048 -new -nodes -x509 -days 3650 -keyout key.pem -out cert.pem
+```
+
+You will be prompted with a few questions after entering the command. Use `127.0.0.1` as value for `Common name` if you want to be able to install the certificate in your OS's root certificate store or browser so that it is trusted.
+
+This generates a cert-key pair and it will be valid for roughly 10 years (3650 days to be exact).
```

Comment:
What do you think of reversing the phrasing here to say

```suggestion
 This generates a cert-key pair and it will be valid for 3650 days (about 10 years).
```

Reply:
Finished